integrated_channels/integrated_channel/admin/__init__.py

```diff
@@ -0,0 +1,92 @@
+"""
+Django admin integration for integrated_channel app.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib import admin
+
+from integrated_channels.integrated_channel.models import (
+    EnterpriseCustomerPluginConfiguration, EnterpriseIntegratedChannel
+)
+
+
+@admin.register(EnterpriseIntegratedChannel)
+class EnterpriseIntegratedChannelAdmin(admin.ModelAdmin):
+    """
+    Django admin model for EnterpriseIntegratedChannel.
+    """
+    list_display = (
+        "name",
+        "data_type",
+    )
+
+    list_filter = ("data_type",)
+    search_fields = ("name", "data_type",)
+
+    class Meta(object):
+        model = EnterpriseIntegratedChannel
+
+
+@admin.register(EnterpriseCustomerPluginConfiguration)
+class EnterpriseCustomerPluginConfigurationAdmin(admin.ModelAdmin):
+    """
+    Django admin model for EnterpriseCustomerPluginConfiguration.
```

Comment:
@brittneyexline After seeing your demo today, it made me question if we even need this model?

We already have `EnterpriseIntegratedChannel` to determine what channels are enabled or not.

Then we also have models like `SAPSuccessFactorsEnterpriseCustomerConfiguration` which can link each particular enabled channel to the `EnterpriseCustomer`.

So if you add an `active` flag to `SAPSuccessFactorsEnterpriseCustomerConfiguration`, then `EnterpriseCustomerPluginConfiguration` seems completely redundant. I think it might even be unhelpful to have - after all, what would it mean if GriffCo is enabled for SAP according to `EnterpriseCustomerPluginConfiguration` but there is no corresponding `SAPSuccessFactorsEnterpriseCustomerConfiguration` record for GriffCo? If you remove the intermediate model then you can't have that kind of inconsistent state in the database. (Which is especially important when some of these links between models are not enforced by foreign keys.)


This is very analogous to [the models in `third_party_auth`](https://github.com/edx/edx-platform/blob/f806f3a/common/djangoapps/third_party_auth/models.py#L79) - there, I put the common functionality (similar to `EnterpriseCustomerPluginConfiguration`) into an [abstract base class](https://docs.djangoproject.com/en/1.10/topics/db/models/#abstract-base-classes) (which includes, for example, the `enabled` flag, analogous to the `active` flag), and I made each type of provider inherit from that abstract base class.